src/hyp3_isce2/insar_stripmap.py

```diff
@@ -0,0 +1,75 @@
+"""
+ISCE2 processing
+"""
+
+import argparse
+import logging
+import os
+import site
+import sys
+from pathlib import Path
+
+from hyp3lib.aws import upload_file_to_s3
+from hyp3lib.image import create_thumbnail
+
+
+log = logging.getLogger(__name__)
+
+# ISCE needs its applications to be on the system path.
+# See https://github.com/isce-framework/isce2#setup-your-environment
+ISCE_APPLICATIONS = Path(site.getsitepackages()[0]) / 'isce' / 'applications'
+if str(ISCE_APPLICATIONS) not in os.environ['PATH'].split(os.pathsep):
+    os.environ['PATH'] = str(ISCE_APPLICATIONS) + os.pathsep + os.environ['PATH']
+
+
+def insar_stripmap(
+    reference_scene: str,
+    secondary_scene: str,
+) -> Path:
```

Comment:
```suggestion
def insar_stripmap(reference_scene: str, secondary_scene: str) -> Path:
```
We can shorten this line now